tests/e2e/conf/prod-us-east.conf

```diff
@@ -69,3 +69,10 @@ TEST_SUITE_RUN=basic
 # Testcase variables
 PLUGIN_NAME=ibm.io/ibmc-block
 PVC_ACCESS_MODE=RWO
+
+# Volume creation varibles
+OLD_REQUEST_URL=containers.bluemix.net
+NEW_REQUEST_URL=containers.bluemix.net
+OLD_REGION="region: us-east"
+NEW_REGION="region: $ARMADA_REGION"
```

Comment:
You dont need the `$ARMADA_REGION` variable. Instead you can set the region as `us-east` here as this conf file is specific to us-east region.

Reply:
That was old variable being used to create cluster and other operations using the same